src/LandingPage/CreateUserModal.js

```diff
@@ -1,32 +1,322 @@
+        this.onSubmit = this.onSubmit.bind(this);
+    };
+
+    componentDidMount() {
+        this.props.fetchMeetingOptions();
+    }
+
+    renderOptionItems() {
+        return this.props.allMeetingOptions.map(item =>
+            (
+                <option
+                    key={item.id}
+                    value={item.id}
+                    checked={
+                        this.state.user.meet_options &&
+                        this.state.user.meet_options.indexOf(item.id) > -1
+                    }
+                >
+                {item.name}
+                </option>
+            )
+        );
+    }
+
+    selectionRenderer(values) {
+        switch (values.length) {
+            case 0:
+                return "";
+            case 1:
+                return this.props.allMeetingOptions[values[0] - 1].name;
+            default:
+                return `${values.length} options selected`;
+        }
+    };
+
+    handleChange(event) {
+        const { user } = this.state;
+        user[event.target.name] = event.target.value;
+        this.setState({ user });
+    }
+
+    onClick() {
+        const input = document.getElementById("input");
+        input.click();
+    }
+
+    renderImg() {
+        if (this.state.imgURL) {
+            return (
+                <div>
+                    <img
+                        className="img img-thumbnail"
+                        src={this.state.imgURL}
+                        alt=""
+                        width="250"
+                        height="250"
+                    />
+                </div>
+            );
+        } else {
+            return (
+                <div>
+                    <img
+                        className="img img-thumbnail"
+                        src='../images/holder.jpg'
+                        alt=""
+                        width="250"
+                        height="250"
+                    />
+                </div>
+            );
         }
+    }
+
+    onUpload(event) {
+        const file = event.target.files;
+        const fileReader = new FileReader();
+
+        fileReader.readAsDataURL(file[0]);
+        fileReader.onload = () => {
+            this.setState({ imgURL: fileReader.result, img: file[0] });
+        };
+    }
+
+    storeProfilePicture() {
+        const ext = this.state.img.name.slice(this.state.img.name.lastIndexOf('.'));
+        return firebase.storage()
+            .ref(`/users/${this.state.user.first_name}${this.state.user.last_name}${ext}`)
+            .put(this.state.img)
+            .then(snapshot => {
+                this.state.user.photo = snapshot.downloadURL;
+            });
     };
 
+    onSubmit(e) {
+        e.preventDefault();
+        console.log(this.state.user.email);
+        let user = this.state.user
+        let { first_name, last_name, email, password, city, experience, training, video_link, website } = user
+
+        if (this.state.imgURL.startsWith("data:image")) {
```

Comment:
Is this going to require users to have a profile picture? Can we check to make sure there's an imgURL at all, and then check to see the prefix on it?

Reply:
done